src/components/list-item/list-item.tsx

```diff
@@ -281,12 +281,11 @@ export class ListItem implements ConditionalSlotComponent, InteractiveComponent
   }
 
   render(): VNode {
-    const { expandable, expanded, level, posInSet, setSize, active, label, description } = this;
+    const { expandable, expanded, level, posInSet, setSize, active } = this;
     return (
       <Host>
         <tr
           aria-expanded={expandable ? toAriaBoolean(expanded) : null}
-          aria-label={label}
```

Comment:
We'll need to keep the `aria-label` in the `<tr>`, otherwise no announcement exists.